plugins/destination/plugin_testing.go

```diff
@@ -253,3 +230,31 @@ func PluginTestSuiteRunner(t *testing.T, p *Plugin, spec any, tests PluginTestSu
 		}
 	})
 }
+
+func createTestResources(table *schema.Table, sourceName string, syncTime time.Time, count int) []schema.DestinationResource {
+	resources := make([]schema.DestinationResource, count)
+	for i := 0; i < count; i++ {
+		resource := schema.DestinationResource{
+			TableName: table.Name,
+			Data:      testdata.GenTestData(table),
+		}
+		_ = resource.Data[0].Set(sourceName)
+		_ = resource.Data[1].Set(syncTime)
+		resources[i] = resource
+	}
+	return resources
+}
+
+func sortResources(table *schema.Table, resources []schema.DestinationResource) {
+	cqIDIndex := table.Columns.Index(schema.CqIDColumn.Name)
+	sort.Slice(resources, func(i, j int) bool {
```

Comment:
```suggestion
	sort.SliceStable(resources, func(i, j int) bool {
```

Reply:
It's a two-element slice being sorted on UUIDs, I don't think it really maters 😄